scalableminds/webknossos
# Refactor RPC Class/Variable Naming

- Extracted the wk master token to separate object RpcTokenHolder
- Renamed some variables and classes concerning RPCs between webKnossos and the datastore + tracingstore modules
- New naming scheme examples: “DSRemoteWebKnossosClient” lives in the datastore, and sends RPC calls to webKnossos. “WKRemoteDataStoreController” lives in webKnossos and handles RPC requests sent by the data store.
- The variable name `webKnossosServer` was ditched, I’d rather consider the client as such, rather than think of it as a wrapper around the remote server.
- The naming scheme is of course open for debate, I’m looking forward to your opinions :)

### URL of deployed dev instance (used for testing):
- https://refactorrpcnames.webknossos.xyz

### Steps to test:
- Should still be able to create/save annotation, upload/download NML, edit dataset settings (e.g. bbox)

### Issues:
- fixes #5275 

----
